src/graph.js

```diff
@@ -108,11 +108,37 @@ class Graph {
 	 * @param {Map} [params] Parameters map
 	 * @returns {Promise<ResultSet>} a promise contains a result set
 	 */
-	async query(query, params) {
+	query(query, params) {
+		return this._query("graph.QUERY", query, params);
+	}
+
+	/**
+	 * Execute a Cypher readonly query
+	 * @async
+	 * @param {string} query Cypher query
+	 * @param {Map} [params] Parameters map
+	 *
+	 * @returns {Promise<ResultSet>} a promise contains a result set
+	 */
+	readonlyQuery(query, params) {
```

Comment:
Shouldn't you mark it as async too?
```suggestion
	async readonlyQuery(query, params) {
```

Reply:
using the `async` keyword will create another promise